pulsar-sql/presto-pulsar/src/test/java/org/apache/pulsar/sql/presto/TestPulsarConnector.java

```diff
@@ -179,28 +150,13 @@
     public static class Bar {
         public Integer field1;
         public String field2;
-        public Boo test;
```

Comment:
Is there any test to test the nest object query?

Reply:
Yes, `Foo.bar` tested in main modules, and `TestAvroDecoder\TestJsonDecoder` has also test nest object in Decoder.